models/classes/class.QtiTestService.php

```diff
@@ -866,24 +869,62 @@ protected function importTest(
         return $report;
     }
 
+    /**
+     * @throws common_Exception
+     */
     private function deleteTestsFromClassByLabel(
         string $testLabel,
         string $itemsClassLabel,
         core_kernel_classes_Resource $testClass,
         core_kernel_classes_Class $itemClass
     ): void {
-        $testService = $this->getTestService();
-        $itemTreeService = $this->getItemTreeService();
+        $itemClassesToDelete = $this->getSubclassesByLabel($itemClass, $itemsClassLabel);
+
+        /** @var string[] $refs */
+        $refs = $this->collectResourceReferences($itemClassesToDelete);
 
+        /** @var string[] $deletedTestsUris */
+        $deletedTestsUris = [];
```

Comment:
Why type hint local variables? Is it sustainable to do it?

```suggestion
        $deletedTestsUris = [];
```

Reply:
Thanks!